packages/js/src/ui/components/Inbox.tsx

```diff
@@ -1,14 +1,14 @@
-import { Accessor, createSignal, JSX, Match, Switch } from 'solid-js';
+import { createSignal, Match, Switch } from 'solid-js';
 import { useStyle } from '../helpers';
-import { NotificationMounter } from '../types';
+import { BellMounter, NotificationMounter } from '../types';
 import { Bell, Footer, Header, Settings, SettingsHeader } from './elements';
 import { NotificationList } from './Notification';
 import { Button, Popover } from './primitives';
 
 export type InboxProps = {
   open?: boolean;
-  mountNotification?: NotificationMounter;
-  renderBell?: ({ unreadCount }: { unreadCount: Accessor<number> }) => JSX.Element;
+  renderNotification?: NotificationMounter;
```

Comment:
❓ Why are we changing the name here?

Reply:
Wanted to align the naming throughout the app
let me know if you think we shouldn't update this

Reply:
Yeah I think it's better to leave as `mountNotification`, In the JS level, it is a mount method, just like the `mountInbox`.

Reply:
So what we can do is to have `renderNotification, renderBell` as the props on the `react` package, but map it internally to the `mountNotification, mountBell` functions in the JS SDK.